hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/SubscriptionService.java

```diff
@@ -120,6 +126,13 @@ public SubscriptionMetrics getSubscriptionMetrics(TopicName topicName, String su
         return metricsRepository.loadMetrics(topicName, subscriptionName);
     }
 
+    public SubscriptionHealth getSubscriptionHealth(TopicName topicName, String subscriptionName) {
+        Subscription subscription = getSubscriptionDetails(topicName, subscriptionName);
+        TopicMetrics topicMetrics = topicService.getTopicMetrics(topicName);
```

Comment:
This is very counter intuitive. We have certain metrics available via zookeeper so if we try to get metrics for non existing topic then zookeeper would create empty nodes for us which is a problem because for instance listing available topics is dependent on zookeeper tree.